src/models/organization.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Organization extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
```

Comment:
Acá es conveniente declarar la relación entre Organization y Links para después poder obtener los links directamente desde el modelo Organization con una query del tipo: `Organization.findAll({ include: Links });`.

```js
static associate(models) {
  Organization.hasMany(models.Links, {
      foreignKey: 'organizationId'
  });
}
```
- https://sequelize.org/master/manual/eager-loading.html#fetching-all-associated-elements
- https://sequelize.org/v6/manual/assocs.html#one-to-many-relationships

Reply:
Gracias!!! ahora hacemos una sola llamada a la base:

``` js
    const organization = await Organization.findByPk(
      process.env.ORGANIZATION_ID,
      {
        include: Links,
      }
    );
```